extensions/blocks/map/edit.js

```diff
@@ -185,6 +212,51 @@ class MapEdit extends Component {
 						] }
 					/>
 					<PanelBody title={ __( 'Map Settings', 'jetpack' ) }>
+						<BaseControl
+							label={ __( 'Height in pixels' ) }
+							id={ `block-jetpack-map-height-input-${ instanceId }` }
+						>
+							<input
+								type="number"
+								id={ `block-jetpack-map-height-input-${ instanceId }` }
+								onChange={ event => {
+									let height = parseInt( event.target.value, 10 );
+									if ( isNaN( height ) ) {
+										// Set map height to default size and input box to empty string
+										height = null;
+									} else if ( height < 400 ) {
+										// Set map height to minimum size
+										height = 400;
+									}
+									setAttributes( {
+										mapHeight: height,
+									} );
+									setTimeout( this.mapRef.current.sizeMap, 0 );
+								} }
+								value={ mapHeight || '' }
+								min="400"
+								step="10"
+							/>
+						</BaseControl>
+						<RangeControl
+							label={ __( 'Zoom level', 'jetpack' ) }
+							help={
+								points.length > 1
+									? __(
+											'The default zoom level cannot be changed when there are two or more markers on the map.',
+											'jetpack'
+									  )
+									: ''
```

Comment:
We might simplify this as `points.length > 1 && __( 'The default...' )`

Reply:
👍🏻 07b5cc0aa

Reply:
👍🏻 07b5cc0aa